src/memcached.c

```diff
@@ -285,6 +297,47 @@ static void submit_gauge2(const char *type, const char *type_inst,
   plugin_dispatch_values(&vl);
 }
 
+static gauge_t calculate_rate(gauge_t part, gauge_t total, gauge_t *prev_part,
```

Comment:
I realize you copied this from the previous implementation, but "rate" is incorrect and confusing. What you're calculating here is a *ratio*.

Reply:
Renamed.